go/adbc/driver/snowflake/connection.go

```diff
@@ -281,11 +281,14 @@ func (c *cnxn) getObjectsDbSchemas(ctx context.Context, depth adbc.ObjectDepth,
 	}
 
 	conditions := make([]string, 0)
+	queryArgs := make([]interface{}, 0, 2)
```

Comment:
Is there a specific reason to use `interface` here, when the type is know to be `string`?

Reply:
Query arguments only accepts interface